src/compression.rs

```diff
@@ -134,6 +142,8 @@ impl CompressionMethod {
             CompressionMethod::Zstd => 93,
             #[cfg(feature = "lzma")]
             CompressionMethod::Lzma => 14,
+            #[cfg(feature = "xz")]
+            CompressionMethod::Xz => 95,
```

Comment:
Add this to `fuzz/fuzz.dict` in 16-bit little-endian form. That makes it more likely to be covered by the fuzz tests.